src/migrations/versions/0820dad794e6_delete_removed_status.py

```diff
@@ -0,0 +1,46 @@
+"""Delete removed status
+Revision ID: 0820dad794e6
+Revises: f552a27b4390
+Create Date: 2024-10-08 22:41:38.868185
+"""
+from alembic import op
+from alembic_postgresql_enum import TableReference
+
+# revision identifiers, used by Alembic.
+revision = "0820dad794e6"
+down_revision = "f552a27b4390"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("UPDATE job SET status = 'cancelled' WHERE status = 'removed';")
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.sync_enum_values(
+        "public",
+        "jobstatus",
+        ["done", "new", "cancelled", "processing"],
+        [
+            TableReference(
+                table_schema="public", table_name="job", column_name="status"
+            )
+        ],
+        enum_values_to_rename=[],
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.sync_enum_values(
+        "public",
+        "jobstatus",
```

Comment:
actually I don't care about downgrates too much, and it's not even possible in this case. You can just raise NotImplementedError() here.